src/ahbicht/validation/maus_provider.py

```diff
@@ -0,0 +1,67 @@
+"""
+MAUS s' (plural of MAUS) are data structures which are structurally equivalent to the maus.DeepAnwendungshandbuch.
+For more information see the MAUS repo and its documentation: https://github.com/Hochfrequenz/mig_ahb_utility_stack/
+A MAUS Provider is a class that returns MAUS s' from what ever data source the implementation prefers.
```

Comment:
Das Apostroph gibt es nur, wenn es possessiv (?) ist.
The two Maus s' whiskers are very nice.
Many Maus s have many whishers. 

Oder so ähnlich 😅